test/v6/ukPropertyBsas/submit/def1/model/request/UkPropertySpec.scala

```diff
@@ -31,37 +31,37 @@ class NonFurnishedHolidayLetSpec extends UnitSpec {
       |}
       |""".stripMargin)
 
-  private val nonFurnishedHolidayLet = NonFurnishedHolidayLet(
-    Some(NonFHLIncome(None, None, None, None)),
```

Comment:
Maybe use named arguments. 

(Default values probably don't make sense in cases where the production code creates instances exclusively from JSON. Also it could reduce compile-time safety: e.g. if another field is added, compilation succeeds for call sites when maybe it shouldn't.)